packages/gatsby-recipes/src/create-types.js

```diff
@@ -2,6 +2,7 @@ const Joi2GQL = require(`./joi-to-graphql`)
 const Joi = require(`@hapi/joi`)
 const { GraphQLString, GraphQLObjectType, GraphQLList } = require(`graphql`)
 const _ = require(`lodash`)
+const { ObjectTypeComposer, schemaComposer } = require(`graphql-compose`)
```

Comment:
this is a convenient helper function!

Reply:
Super nice! Glad I stumbled upon it, shoutout to @freiksent who pointed it out in some really old graphql.js issue :fire: